python/thinwalltopography/merge_topog_tiles.py

```diff
@@ -2,7 +2,7 @@
 
 import sys, getopt
 import datetime, os, subprocess
-import GMesh
+from thinwalltopography import GMesh
```

Comment:
This approach has the drawback that tools like this cannot be called like tools anymore, I.e., 
```
./merge_topog_tiles.py 
Traceback (most recent call last):
  File "./merge_topog_tiles.py", line 5, in <module>
    from thinwalltopography import GMesh
ModuleNotFoundError: No module named 'thinwalltopography'
```

Reply:
They can only be imported in external modules or notebooks.

Reply:
have you run ```python setup.py install``` ?

we can also add the various scripts to the $PATH in the setup.py

Reply:
```
python setup.py install
running install
error: can't create or remove files in install directory

The following error occurred while trying to add or remove files in the
installation directory:

    [Errno 30] Read-only file system: '/usr/local/python/3.6.4/lib/python3.6/site-packages/test-easy-install-5875.write-test'

The installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    /usr/local/python/3.6.4/lib/python3.6/site-packages/

Perhaps your account does not have write access to this directory?  If the
installation directory is a system-owned directory, you may need to sign in
as the administrator or "root" account.  If you do not have administrative
access to this machine, you may wish to choose a different installation
directory, preferably one that is listed in your PYTHONPATH environment
variable.

For information on other options, you may wish to consult the
documentation at:

  https://setuptools.readthedocs.io/en/latest/easy_install.html

Please make the appropriate changes for your system and try again.
```

Reply:
This was not designed to be a package for users to install, it's just a set of tools.

Reply:
It looks like you are using gfdl's python module, right?
if so, it won't let you install in a protected directory but you can either:

1. install in a 'user' space: ```python setup.py install --user```
2. Create a virtualenv
3. Manage your environments with conda